content/hi/docs/tasks/tools/install-kubectl-windows.md

```diff
@@ -22,12 +22,12 @@ Windows पर kubectl संस्थापित करने के लिए
 
 ### Windows पर curl के माध्यम से kubectl बाइनरी इंस्टॉल करें
 
-1. [latest release {{< param "fullversion" >}}](https://dl.k8s.io/release/{{< param "fullversion" >}}/bin/windows/amd64/kubectl.exe) डाउनलोड करें।
+1. [latest release {{< skew currentPatchVersion >}}](https://dl.k8s.io/release/v{{< skew currentPatchVersion >}}/bin/windows/amd64/kubectl.exe) डाउनलोड करें।
```

Comment:
Small nit otherwise LGTM
```suggestion
1. [नवीनतम रिलीज {{< skew currentPatchVersion >}}](https://dl.k8s.io/release/v{{< skew currentPatchVersion >}}/bin/windows/amd64/kubectl.exe) डाउनलोड करें।
```

Reply:
That fix doesn't need to happen in this PR (it doesn't relate to the change that the PR title describes)